src/components/Home/Home.tsx

```diff
@@ -0,0 +1,223 @@
+
+const Home = () => {
+  return (
+    <>
+      <main>
+        <section className="flex flex-col justify-center items-center pt-[230px] pb-[63px] bg-white">
+          <h1 className="text-center text-[60px] font-bold leading-[80px] whitespace-pre-line mb-[26px]">
+            {'자기소개서가 막막한 순간,\n그 시작을 함께하는\n인사이트 아웃'}
+          </h1>
+          <p className="text-center whitespace-pre-line mb-[52px]">
+            {
+              '인사이트 아웃을 통해 질문으로 나의 프로젝트를 정리하며 경험을 파헤치고\n나만의 경험 역량 카드로 만들어 자기소개서를 작성해보세요.'
+            }
+          </p>
+          <StartButton route={ROUTES.EXPERIENCE}>경험 분해 시작하기</StartButton>
+        </section>
+        <section className="relative h-[1014px] bg-white">
+          <Image src={'/images/home/img-home-1.png'} fill alt="Home-1" className="object-contain w-full h-full" />
+        </section>
+        <section className="relative flex flex-col justify-center items-center pt-[170px] pb-[283px] bg-white z-[-1]">
+          <div className="flex justify-center items-center mb-[286px] gap-[102px]">
+            <div className="flex flex-col gap-[44px]">
+              <div className="flex flex-col w-[588px] gap-[30px]">
+                <Badge variant="primary50-outline" size="L">
+                  경험분해 4 STEPS
+                </Badge>
+                <h1 className="text-main text-[44px] font-bold whitespace-pre-line">
+                  {'질의응답으로\n 경험을 분해해요'}
+                </h1>
+                <div className="flex flex-col gap-[16px]">
+                  <p className="text-light text-[16px] font-medium whitespace-pre-line">
+                    {
+                      '자기소개서 쓰기 전 경험분해를 통해 나의 직무 역량과 강점을 파악하고\n 면접으로 이어지는 것까지 고려하여 직무 강점을 어필할 수 있는 경험으로 분해 해야합니다.'
+                    }
+                  </p>
+                  <p className="text-light text-[16px] font-medium whitespace-pre-line">
+                    {
+                      '인사이트 아웃을 통해 질문으로 나의 이력, 프로젝트를 정리하며 경험을 파헤치고\n나만의 경험 역량 카드로 만들어 자기소개서를 작성해보세요.'
+                    }
+                  </p>
+                </div>
+              </div>
+              <div className="flex items-center w-[612px] py-[24px] border-y-[1px] border-solid border-gray-200 gap-[14px]">
+                {ANALYZE_CARD.map((title, index) => (
+                  <>
+                    <AnalyzeCard key={title} index={index + 1} title="직무 선택" />
+                    {index < 3 && <SvgIconChevronRight />}
+                  </>
+                ))}
+              </div>
+            </div>
+            <Image src={'/images/home/img-home-2.png'} width={486} height={486} alt="Home-2" />
+          </div>
+          <div className="flex flex-col items-center gap-[30px] w-[588px] mb-[170px]">
+            <Badge variant="primary50-outline" size="L">
+              AI 역량키워드 추천
+            </Badge>
+            <p className="text-center whitespace-pre-line h2">
+              {'인공지능이 내 경험과 어울리는\n직무 키워드를 뽑아줘요'}
+            </p>
```

Comment:
```suggestion
            <h2 className="text-center whitespace-pre-line h2">
              {'인공지능이 내 경험과 어울리는\n직무 키워드를 뽑아줘요'}
            </h2>
```

뱃지 아래에 있는 텍스트들이 각 섹션의 제목인 것 같은데 h2로 사용하는건 어떠세요?!

Reply:
좋습니다~